recaptcha_enterprise/snippets/create_mfa_assessment.py

```diff
@@ -0,0 +1,115 @@
+
+    Args:
+        project_id: GCloud Project ID
+        recaptcha_site_key: Site key obtained by registering a domain/app to use recaptcha services.
+        token: The token obtained from the client on passing the recaptchaSiteKey.
+            To get the token, integrate the recaptchaSiteKey with frontend. See,
+            https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages#frontend_integration_score
+        recaptcha_action: The action name corresponding to the token.
+        hashed_account_id: Create hashedAccountId from user identifier.
+            It's a one-way hash of the user identifier: HMAC SHA 256 + salt
+        email: Email id of the user to trigger the MFA challenge.
+        phone_number: Phone number of the user to trigger the MFA challenge. Phone number must be valid
+            and formatted according to the E.164 recommendation.
+            See: https://www.itu.int/rec/T-REC-E.164/en
+    """
+
+    client = recaptchaenterprise_v1.RecaptchaEnterpriseServiceClient()
+
+    # Set the properties of the event to be tracked.
+    event = recaptchaenterprise_v1.Event(
+        site_key=recaptcha_site_key,
+        token=token,
+        hashed_account_id=hashed_account_id
+    )
+
+    # Set the email address and the phone number to trigger/ verify the MFA challenge.
+    endpoint_verification_info = recaptchaenterprise_v1.EndpointVerificationInfo(
+        email_address=email,
+        phone_number=phone_number
+    )
+
+    account_verification_info = recaptchaenterprise_v1.AccountVerificationInfo(
+        endpoints=[endpoint_verification_info]
+    )
+
+    assessment = recaptchaenterprise_v1.Assessment(
+        event=event,
+        account_verification=account_verification_info
+    )
+
+    project_name = f"projects/{project_id}"
+
+    # Build the assessment request.
+    request = recaptchaenterprise_v1.CreateAssessmentRequest(
+        assessment=assessment,
+        parent=project_name
+    )
+
+    # Check integrity of the response.
+    response = client.create_assessment(request)
+    if not verify_response_integrity(response, recaptcha_action):
+        raise RuntimeError("Failed to verify token integrity.")
+
+    result = response.account_verification.latest_verification_result
+    # If the result is unspecified, send the request token to trigger MFA in the client.
+    # You can choose to send both the email and phone number's request token.
+    if result == recaptchaenterprise_v1.types.AccountVerificationInfo.Result.RESULT_UNSPECIFIED:
+        print("Result unspecified. Trigger MFA challenge in the client by passing the request token.")
+        # Send the request token for assessment. The token is valid for 15 minutes.
+        # print(response.account_verification.endpoints[0].request_token)
+
+    # If the result is not unspecified, return the result.
+    print(f"MFA result: {result}")
+
+
+def verify_response_integrity(response: Assessment, recaptcha_action: str) -> bool:
```

Comment:
update this along with suggestion above to remove the direct class import.
```suggestion
def verify_response_integrity(
    response: recaptchaenterprise_v1.Assessment,
    recaptcha_action: str
) -> bool:
```